### PR TITLE
docs: import the latest event info to Events page

### PR DESCRIPTION
**Pull request by Boburmirzo**
Fixes: #[Add issue number here]

Changes:

Add upcoming events information imported from the internal Lark excel file.
Here, you can see only accepted , confirmed and APISIX related events list.

Screenshots of the change:

image

### CODE REVIEW

File: website/docs/general/events.md [hunk 1]
```diff
@@ -18,16 +18,105 @@ that have a specific focus on Apache APISIX.
...
+
 #### [CloudLand](https://www.cloudland.org/de/home/)
 
-* **Topic:** Evolving your APIs
-* **Speaker:** [Nicolas Fränkel](https://github.com/nfrankel)
-* **Dates:** from 2022-06-29 to 2022-07-02
-* **Location:** Berlin, Germany
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 06-29-22 to 07-02-22
```
Comment (leslie-tsang) [inline website/docs/general/events.md]: Shall we use the old time format ? the new one looks more confusing to me at a glance.

The rest look good to me. :)

Comment (bisakhmondal) [reply website/docs/general/events.md]: Agree+1
How about going with `DD-MMM-YYYY` format?
For example (29-JUN-2022 to 02-JUL-2022). People don't have to worry if the date follows (DD-MM-YYYY or the current MM-DD-YYYY ) format.

**Review by bisakhmondal — COMMENTED** [review]
In general LGTM. Congratulations on an incoming EU tour and helping APISIX reach out to a larger set of audience : )

File: website/docs/general/events.md (hunk 1 above)
Comment (Boburmirzo) [reply website/docs/general/events.md]: @leslie-tsang and @bisakhmondal 
 Thank you for the review. I have just noticed that my excel formula generated dates in wrong format, sorry:)
 
 But I fixed it and liked this format more 29-JUN-2022 to 02-JUL-2022

**Review by SkyeYoung — DISMISSED** [review]
LGTM

File: website/docs/general/events.md [hunk 2]
```diff
@@ -18,16 +18,105 @@ that have a specific focus on Apache APISIX.
...
+* **Topic** Chopping the monolith
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 29-June-2022 to 02-July-2022
+* **Location** Germany
+
+#### [Web Developer Conference](https://www.web-developer-conference.de/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 26-September-2022 to 28-September-2022
```
Comment (leslie-tsang) [inline website/docs/general/events.md]: Please keep consistent abbreviations.

Use `Sep` instead of `September `

Comment (Boburmirzo) [reply website/docs/general/events.md]: Good point! I fixed it too.

**Review by juzhiyuan — APPROVED** [review]
LGTM, welcome more events from community!

**Review by SkyeYoung — APPROVED** [review]

**Review by bisakhmondal — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**